public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -474,7 +475,10 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange, mod
         }
       : undefined;
 
-  const hasDatasourcesForDataSourceManaged = Boolean(rulesSourcesWithRuler.length);
```

Comment:
Didn't really need to do the previous PR, could have just gone straight to this 😁